### ISSUE TITLE
jooq-meta-3.5.0.xsd is not valid XSD 1.0

### ISSUE THREAD

**Issue by tazle** [jOOQ/jOOQ#4017]
Is jooq-meta-3.5.0.xsd intentionally only valid as XSD 1.1?

It's a bit problematic with IDEA, since it validates the XSD against XSD 1.0 schema, and doesn't validate an XML document that claims to conform to jooq-meta-3.5.0.xsd correctly.

**Comment by lukaseder** [comment]
Interesting. To be honest, we never checked, explicitly. We're using `XSD` for two reasons:

- To formally communicate the API
- To generate Java classes using XJC

I suspect, the invalid XSD is about replacing:

```xml
    <complexType name="Schemata">
        <all>
            <element name="schema" type="tns:Schema" minOccurs="0" maxOccurs="unbounded" />
        </all>
    </complexType>
```

by...

```xml
    <complexType name="Schemata">
        <sequence>
            <element name="schema" type="tns:Schema" minOccurs="0" maxOccurs="unbounded" />
        </sequence>
    </complexType>
 ```

**Comment by lukaseder** [comment]
This fix will be merged to jOOQ 3.5.2: #4018
